projects/aca-playwright-shared/src/api/shared-links-api.ts

```diff
@@ -48,4 +50,50 @@ export class SharedLinksApi {
       return null;
     }
   }
+
+  async shareFilesByIds(ids: string[]): Promise<SharedLinkEntry[]> {
+    const sharedLinks: SharedLinkEntry[] = [];
+    try {
+      if (ids && ids.length > 0) {
+        for (const id of ids) {
+          const sharedLink = await this.shareFileById(id);
+          sharedLinks.push(sharedLink);
+        }
+      }
+    } catch (error) {
+      logger.error(`SharedLinksApi shareFilesByIds : catch : `, error);
+    }
+    return sharedLinks;
+  }
+
+  private async getSharedLinks(maxItems: number = 250): Promise<SharedLinkPaging | null> {
```

Comment:
I don't like `| null` pattern that you're using here, you can just return empty object here and check it later without adding `null` as additional return type

Reply:
corrected Thanks.